### ISSUE TITLE
ST_AsGeoJSON not working the same > 0.7.0

### ISSUE THREAD

**Issue by sachsbl** [geoalchemy/geoalchemy2#278]
Noticed a potential bug/issue today when upgrading to 0.8.3.  Here's the code:

`query = session.query(func.ST_AsGeoJSON(func.ST_Collect(MyModel.geom)))`

 "geom" column is simple Geoalchemy2 Geometry (`geom = Column(Geometry)`)

for this example lets say the schema is `my_schema` and the table is `my_table`
0.7.0 gives:
```
SELECT ST_AsGeoJSON(ST_Collect(my_schema.my_schema.geom)) AS "ST_AsGeoJSON_1" 
FROM my_schema.my_schema
```

0.8.3 gives:
```
SELECT ST_AsGeoJSON(ST_Collect(my_schema) AS "ST_AsGeoJSON_1" 
FROM ST_Collect(my_schema.my_table.geom)
```
This throws a Syntax Error in PostGIS 2.5x, PostGres 11.7

**Comment by adrien-berchet** [comment]
Hi @sachsbl, thanks for the report.
This error is due to the new internal design and the missing of `ST_Collect` in `geoalchemy2.functions`. The master branch should fix this issue because `ST_Collect` was added.
I think I will release this soon.

**Comment by sachsbl** [comment]
Thanks @adrien-berchet! I will watch for the new release

**Comment by adrien-berchet** [comment]
In the end I realized that there was a bug in the `ST_AsGeoJSON` function. I fixed it in #279 and released the 0.8.4 version.
